packages/core/src/manifest.ts

```diff
@@ -150,6 +229,9 @@ export class Manifest {
       return validateOrUpdateManifestVersion(data);
     } catch (e: any) {
       if (e.code === 'ENOENT') {
+        if (this.parent !== undefined) {
+          return await this.parent.read();
+        }
         return defaultManifest();
```

Comment:
This is subjective, I personally prefer like this:

```suggestion
        } else {
          return defaultManifest();
        }
```